draft-ietf-quic-multipath.md

```diff
@@ -1399,7 +1399,40 @@ TBD (experiments use 0x1001d76d3ded42f3)| MP_PROTOCOL_VIOLATION | Multipath prot
 
 # Security Considerations
 
-TBD
+## Path States
```

Comment:
We are missing some kind of introduction. We need to say that there is basically no changes from RFC 9000, and in particular to the use of TLS, except in four domains:

* Allocation of resource for Connection Identifiers per path, 
* Allocation of several simultaneous path contexts, and associated resource,
* Greater use of path creations, and thus risks of the request forgery attacks
* Use of multiple number spaces, and encryption requirements

Reply:
I had the same feeling about lacking general introduction for this section. Trying to add some texts.